### PR TITLE
Add Portia Bumanlag to Dev Board

### PR DESCRIPTION
**Pull request by MapleSyyrup**

### CODE REVIEW

**Comment by Jansalvador1445** [comment]
This is Good but please put your data on the last part. Thank you!

**Comment by Jansalvador1445** [comment]
Has `Conflict` please rebase and add your data on the last part. Thank you!

**Comment by MapleSyyrup** [comment]
I hope it resolved the conflict. Thank you so much.

**Comment by Jansalvador1445** [comment]
Hi @MapleSyyrup ! Still has a conflict, just merged sa master, can you rebase afgain? thanks! :D

**Comment by MapleSyyrup** [comment]
Hello. Pacheck po kung wala ng conflict. Thank you.

File: assets/data/devs.json [hunk 1]
```diff
@@ -171,7 +171,6 @@
     "roles": ["Software Engineer"],
     "about": "Dev"
   },
-  {
```
Comment (Jansalvador1445) [inline assets/data/devs.json:174]: it could cause an error here. please add `{`

File: assets/data/devs.json [hunk 2]
```diff
@@ -202,5 +201,10 @@
     "roles": ["Web Developer", "Flutter Developer"],
     "about": "Interested in devising a better problem-solving method for challenging tasks",
     "username": "binos30"
-  }
+  },
+  {
+		"name": "Portia Bumanlag",
```
Comment (Jansalvador1445) [inline assets/data/devs.json:206]: Please fix the indention. thanks

**Comment by MapleSyyrup** [comment]
Please check po thank you :)

**Comment by Jansalvador1445** [comment]
this is good, kindly add `username` on your data. in that way, we will not issue another ticket for `username`. 

Please add this  `"username": "your_username"`  

e.g. `"username" : "juan_delacruz"`

**Comment by Jansalvador1445** [comment]
and then don't forget to comment below the issue ticket. thanks

**Comment by MapleSyyrup** [comment]
need po bang name ko din yung ilagay as username?

**Comment by Jansalvador1445** [comment]
Has `conflict` just merged a new PR kaya need rebase uli. Yes need for the image display on the project. Just add this one.

`"username" : "MapleSyyrup"`

**Comment by Jansalvador1445** [comment]
No more conflict, ready for merging. Can you comment on the link to the filed issue? thanks! :D

File: assets/data/devs.json [hunk 3]
```diff
@@ -203,9 +203,16 @@
...
+    "name": "Jan Dranreb Balangue",
+    "roles": ["Aspiring Flutter Developer"],
+    "about": "Dev",
+    "username": "shadowprend15"
+  },
+  {
     "name": "Portia Bumanlag",
     "roles": ["Accountant", "Flutter Developer"],
     "about": "Flutter Developer since 2020",
+    "username" : "MapleSyyrup"
```
Comment (MapleSyyrup) [inline assets/data/devs.json]: Add Portia Bumanlag to Dev Board

Comment (MapleSyyrup) [reply assets/data/devs.json]: Tama po ba to? hahaha sorry di pa po ako marunong @Jansalvador1445

**Comment by Jansalvador1445** [comment]
Good Job @MapleSyyrup ! Keep it up! :D Merging this now, adding you as contributor too! 

@all-contributors please add @MapleSyyrup  for content

**Comment by MapleSyyrup** [comment]
Thank you so much po! :D

**Comment by Jansalvador1445** [comment]
You could check your image below, https://github.com/flutterph/devs. Just scroll down up to the bottom. :D 

Congrats again! keep it up! :D

**Pull request merged**